addon/-private/external/task-instance/executor.js

```diff
@@ -6,21 +6,21 @@ import {
   YIELDABLE_THROW,
   YIELDABLE_RETURN,
   YIELDABLE_CANCEL,
-  cancelableSymbol
-} from '../yieldables';
+  cancelableSymbol,
+} from "../yieldables";
```

Comment:
I noticed this too when working in the repo -- it might be nice to add Prettier to the linter config. It's harder working in repos where it _isn't_ present than to work in ones where it is!

Reply:
Definitely! Big fan of prettier, I just hate setting it up myself 😆